src/rust/ide/enso-protocol/src/language_server.rs

```diff
@@ -130,14 +130,14 @@ trait API {
     fn destroy_execution_context(&self, context_id:ContextId) -> ();
 
     /// Move the execution context to a new location deeper down the stack.
-    #[MethodInput=PushExecutionContextInput,rpc_name="executionContext/push",
-    result=push_execution_context_result,set_result=set_push_execution_context_result]
-    fn push_execution_context(&self, context_id:ContextId, stack_item:StackItem) -> ();
+    #[MethodInput=ExecutionContextPushInput,rpc_name="executionContext/push",
+    result=execution_context_push_result,set_result=set_execution_context_push_result]
+    fn execution_context_push(&self, context_id:ContextId, stack_item:StackItem) -> ();
```

Comment:
And that's confirmation that the naming change is good :) because for me it's not "pushing the execution context" but "push call to the execution context's stack"